qos-monitor/src/main/java/eu/arrowhead/core/qos/service/ping/monitor/PingEventCollectorTask.java

```diff
@@ -99,7 +99,7 @@ private void addEventToBufferElement (final PingEventBufferElement element, fina
 		logger.debug("addEventToBufferElement started...");
 
 		if(element.getEventArray()[position] != null) {
-			logger.warn(element.getId() + REPLACING_MEASURMENT_EVENT + element.getEventArray()[position].toString());
+			logger.warn(element.getId() + REPLACING_MEASUREMENT_EVENT + element.getEventArray()[position].toString());
```

Comment:
None of the child classes of MeasurementMonitoringEvent overrides toString(), and the default implementation only contains an instanceId.

Reply:
done

Reply:
don't forget about another child classes' toString()

Reply:
done